src/app/[locale]/(private)/error.tsx

```
@@ -0,0 +1,14 @@
+'use client';
+
+import { useEffect } from 'react';
+import { useServerErrorToast } from '@/hooks/use-server-error-toast';
+
+export default function Error() {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)